MAUI/DigitalGauge/Customize-character-segments.md

```diff
@@ -0,0 +1,268 @@
+---
+layout: post
+title: Customize Character Segments in Maui Digital Gauge | Syncfusion
+description: Learn here all about Customize Character Segments support in Syncfusion Maui Digital Gauge (SfDigitalGauge) control and more.
+platform: maui 
+control: DigitalGauge
+documentation: ug
+---
+
+# Customize Character Segments in Maui Digital Gauge (SfDigitalGauge)
+
+The characters of a digital gauge can be customized in terms of `CharacterWidth`, `CharacterHeight`, `CharacterSpacing`, and `CharacterStrokeColor`.
+
+## customize character size
+
+The values of digital characters are scaled by altering the height and width of digital characters. It is achieved by setting the `CharacterHeight`and `CharacterWidth` properties in the digital gauge. Default value of `CharacterHeight` and `CharacterWidth` is 25.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <gauge:SfDigitalGauge Text="SYNCFUSION" StrokeWidth="4" 
+                          HeightRequest="120"  WidthRequest="330" 
+                          BackgroundColor="Black"
+                          DisabledSegmentAlpha="0.1" DisabledSegmentStroke="#146CED"
+                          CharacterHeight="117" CharacterWidth="25" 
+                          HorizontalOptions="Center" VerticalOptions="Center"
+                          CharacterType="SevenSegment" 
+                          CharacterStroke="#146CED"/>
+{% endhighlight %}
+
+{% highlight c# %}
+
+            SfDigitalGauge digital = new SfDigitalGauge();
+
+            digital.HeightRequest = 120;
+
+            digital.WidthRequest = 330;
+
+            digital.Text = "SYNCFUSION";
+
+            digital.CharacterHeight = 117;
+
+            digital.CharacterWidth = 25;
+
+            digital.HorizontalOptions = LayoutOptions.Center;
+
+            digital.VerticalOptions = LayoutOptions.Center;
+
+            digital.StrokeWidth = 4;
+
+            digital.CharacterType = DigitalGaugeCharacterType.SevenSegment;
+
+            digital.DisabledSegmentAlpha = 0.1;
+
+            digital.CharacterStroke = Color.FromRgb(20, 108, 237);
+
+            digital.DisabledSegmentStroke = Color.FromRgb(20, 108, 237);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+## Setting character spacing
+
+The values of digital characters are spaced by altering the space of digital characters. It is achieved by setting the `CharacterSpacing` property.
```

Comment:
The values of digital characters are spaced by altering the space between digital characters. This is achieved by setting the `CharacterSpacing` property.